### ISSUE TITLE
[enhancement]: zoom and pan in viewer

### ISSUE THREAD

**Issue by Jonseed** [invoke-ai/InvokeAI#6989]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What should this feature add?

It would be nice to be able to zoom and pan in the viewer mode, especially to see the image at 100%. Currently you can only zoom and pan in the canvas mode.

### Alternatives

Open image in a new tab, and zoom/pan there.

**Comment by psychedelicious** [comment]
Agreed. This has been on my list for a while - basically as soon as I finished with the viewer in its current state.

It's implemented using the DOM, but I'd like to redo it so that it uses a canvas. This would make it a lot easier to support zoom/pan and some other stuff, like tiled preview and better handling of comparison between images w/ different sizes and aspect ratios.

**Comment by Enigmatum** [comment]
This would be a great addition. I keep trying to zoom in to compare details using the mouse scroll wheel in Single Image Viewer, but no :)

**Comment by mholtzhausen** [comment]
I agree. This is a blocker -- specifically, it would be really great to be able to use zoom + compare at the same time.

**Comment by gobshyte** [comment]
Oh wow yes plus one for this, it's a must have really, I thought mine was just broken!
